server/controllers/stock/reports/stock_movement_report.handlebars

```diff
@@ -11,19 +11,14 @@
     <!-- body  -->
     <div class="row">
       <div class="col-xs-12">
-
-    <!-- page title  -->
-        <h2 class="text-center">
-          {{translate 'REPORT.STOCK_MOVEMENT_REPORT.TITLE'}}
-        </h2>
-
-        <h4 class="text-center">
-          {{date dateFrom}} - {{date dateTo}}
-        </h4>
+        <h2 class="text-center">{{translate 'REPORT.STOCK_MOVEMENT_REPORT.TITLE'}}</h2>
```

Comment:
Could you also change the `{{> head TITLE="TREE.STOCK_MOVEMENT_REPORT" }}` to this title key as well?